api/trace/b3_propagator.go

```diff
@@ -59,11 +59,10 @@ func (b3 B3) Inject(ctx context.Context, supplier propagation.HTTPSupplier) {
 	if b3.SingleHeader {
 		sampled := sc.TraceFlags & core.TraceFlagsSampled
 		supplier.Set(B3SingleHeader,
-			fmt.Sprintf("%s-%.16x-%.1d", sc.TraceIDString(), sc.SpanID, sampled))
+			fmt.Sprintf("%s-%s-%.1d", sc.TraceIDString(), sc.SpanID, sampled))
```

Comment:
Is TraceIDString now duplicative?

Reply:
Actually yes. But from [this](https://github.com/open-telemetry/opentelemetry-go/issues/577#issuecomment-604559360) comment from @MrAlias , seems like the duplicate has to serve other use cases. I can remove the duplicate and tests if required. @MrAlias Please let me know.

Reply:
The idea was that `TraceIDString` would return a representation for a protocol format and the `String` method would be the format for any logging efforts.

It looks like they are equivalent in their representation currently. I'm fine with unifying on `String` it we expect that to remain.

\cc @jmacd , @krnowak

Reply:
Removed TraceIDString() and SpanIDString() methods.